bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx127-eu.html

```diff
@@ -0,0 +1,153 @@
+  {% set item_cook = 'Puoi tenere un occhio su una ricetta e un altro sul suo video tutorial mentre cucini.' %}
+  {% set item_entertain = 'Puoi mettere su un video per i tuoi bimbi mentre stai lavorando.' %}
+{% elif LANG == 'pl' %}
+  {% set main_title = 'Multitasking: 1 <br>FOMO: 0'|safe %}
+  {% set tagline_one = 'Jedno oko na Maroko, a drugie, na co tylko chcesz! Z nową funkcją Firefoksa Picture-in-Picture możesz oglądać mecz i pracować w tym samym czasie.' %}
+  {% set tagline_two = 'Wypróbuj tę funkcję z filmikiem powyżej.
+  Naciśnij przycisk odtwarzania i najedź kursorem na wideo. Następnie kliknij w <strong>przycisk „Obraz w obrazie</strong>, aby wyświetlić filmik w okienku pop-up.'|safe %}
+  {% set cta = 'Ustaw Firefoksa jako swoją domyślną przeglądarkę' %}
+  {% set sub_title = 'Oto, jak możesz używać funkcji Picture-in-Picture' %}
+  {% set item_lecture = 'Możesz słuchać wykładu lub brać udział w spotkaniach, robiąc w tym samym czasie notatki' %}
+  {% set item_cook = 'Możesz jednym okiem zaglądać na treść przepisu, a drugim oglądać tutorial, jak przygotować danie, nad którym właśnie pracujesz' %}
+  {% set item_entertain = 'Możesz puszczać bajki dzieciom i pracować w tym samym czasie' %}
+{% else %}
+  {% set main_title = 'Multitasking: 1. <br>FOMO: 0.'|safe %}
+  {% set tagline_one = 'Keep your eye on the ball: with Firefox’s picture-in-picture feature, you can watch games while you work.' %}
+  {% set tagline_two = 'Try it out with the video above. Press play and hover your cursor over the video. Then click <strong>the Picture-in-Picture button</strong> to pop the video out.'|safe %}
+  {% set cta = 'Set Firefox as your default browser' %}
+  {% set sub_title = 'Ways we’ve been using picture-in-picture' %}
+  {% set item_lecture = 'Watching a lecture or meeting while you take notes' %}
+  {% set item_cook = 'Keeping a tutorial video open with a recipe while you cook or bake' %}
+  {% set item_entertain = 'Entertaining cats, dogs and kids while you get work done' %}
+{% endif %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+<section class="wnp-content-main">
+  <div class="mzp-l-content mzp-t-content-md mzp-t-content-nospace">
+
+    {{ video_inline_embed(
+      id='F-nFQryDB0s',
+      title='Introducing the new Firefox... jk. It’s red pandas.',
+      image=resp_img(
+        url='img/firefox/whatsnew/whatsnew127-eu/video-poster.jpg',
+        optional_attributes={
+          'width': '1024',
+          'height': '576'
+        }
+      ),
+    ) }}
+
+    <h2 class="wnp-main-title">{{ main_title }}</h2>
+    <p class="wnp-main-tagline">{{ tagline_one }}</p>
+    <p class="wnp-main-tagline has-pip-icon">{{ tagline_two }}</p>
+
+    <p class="wnp-main-cta"><a class="mzp-c-button mzp-t-product" href="{{ url('firefox.set-as-default.thanks') }}" data-cta-type="button" data-cta-text="Set Firefox as your default browser">{{ cta }}</a></p>
```

Comment:
**Issue:** this will not fire a GA click event without a `data-cta-type` attribute (in addition to `data-cta-text`): https://bedrock.readthedocs.io/en/latest/attribution/0001-analytics.html#generic-ctas

Reply:
Oh sorry it does, my eyes totally missed it. This should be fine as-is.